### PR TITLE
Pulse animation added to the avatar when the user is talking.

### PR DESCRIPTION
**Pull request by BobakOftadeh**
As requested by Tyler, when the user talks there is now a pulse animation around the avatar border.

### CODE REVIEW

File: bigbluebutton-html5/imports/ui/components/user-avatar/styles.scss [hunk 1]
```diff
@@ -49,7 +50,21 @@ $user-list-bg: $color-off-white;
...
+
+@keyframes pulse {
+  0% {
+    box-shadow: 0 0 0 0 $user-color;
+  }
+  70% {
+    box-shadow: 0 0 0 0.5625rem rgba(243, 246, 249, 0);
+  }
+  100% {
+    box-shadow: 0 0 0 0 rgba(243, 246, 249, 0);
```
Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/user-avatar/styles.scss]: Please make this colour a constant and reuse it

File: bigbluebutton-html5/imports/ui/components/user-avatar/styles.scss [hunk 2]
```diff
@@ -49,7 +50,21 @@ $user-list-bg: $color-off-white;
 }
 
 .talking {
-  box-shadow: 0 0 0 1px $user-list-bg, 0 0 0 6px;
+  
+  animation: pulse 1.2s infinite ease-in-out;
```
Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/user-avatar/styles.scss]: In other places in the code we have `animation` combined with `webkit-animation`. Are you sure we don't need to add it here too?

**Comment by antobinary** [comment]
peek 2018-06-01 15-58
Works well!

File: bigbluebutton-html5/imports/ui/components/audio/audio-controls/styles.scss [hunk 3]
```diff
@@ -31,8 +31,32 @@
...
-    span:first-child{
-      box-shadow: 0 0 0 1px #ffffff, 0 0 0 1px;
-    }
+    -webkit-animation: pulse 1s infinite ease-in-out;
+    animation: pulse 1s infinite ease-in-out;
+    border-radius: 4em;
+  }
+}
+
+@-webkit-keyframes pulse {
```
Comment (Tainan404) [inline bigbluebutton-html5/imports/ui/components/audio/audio-controls/styles.scss]: You don't need this, when scss is compiled, the compiler adds the prefix. 

webkit

File: bigbluebutton-html5/imports/ui/stylesheets/variables/palette.scss [hunk 4]
```diff
@@ -24,3 +24,5 @@ $color-gray-label: $color-gray !default;
 $color-blue-light: lighten($color-primary, 35%);
 $color-blue-lighter: #92BCEA;
 $color-blue-lightest: #E4ECF2;
+
+$color-fade: rgba(243, 246, 249, 0);
```
Comment (Tainan404) [inline bigbluebutton-html5/imports/ui/stylesheets/variables/palette.scss]: Why not convert to hexadecimal?

File: bigbluebutton-html5/imports/ui/components/audio/audio-controls/styles.scss [hunk 5]
```diff
@@ -31,8 +31,32 @@
   }
 
   &.glow {
-    span:first-child{
-      box-shadow: 0 0 0 1px #ffffff, 0 0 0 1px;
-    }
+    -webkit-animation: pulse 1s infinite ease-in-out;
+    animation: pulse 1s infinite ease-in-out;
+    border-radius: 4em;
```
Comment (Tainan404) [inline bigbluebutton-html5/imports/ui/components/audio/audio-controls/styles.scss]: there is a border-radius variable in general.sccs that should be used here

File: bigbluebutton-html5/imports/ui/components/audio/audio-controls/styles.scss [hunk 6]
```diff
@@ -31,8 +31,32 @@
   }
 
   &.glow {
-    span:first-child{
-      box-shadow: 0 0 0 1px #ffffff, 0 0 0 1px;
-    }
+    -webkit-animation: pulse 1s infinite ease-in-out;
+    animation: pulse 1s infinite ease-in-out;
```
Comment (Tainan404) [inline bigbluebutton-html5/imports/ui/components/audio/audio-controls/styles.scss]: Is not it better to use ease-in? with ease-in-out it seems that the user has stopped talking.

File: bigbluebutton-html5/imports/ui/stylesheets/variables/palette.scss (hunk 4 above)
Comment (BobakOftadeh) [reply bigbluebutton-html5/imports/ui/stylesheets/variables/palette.scss]: We can't store the alpha in hexadecimal, and eight-digit color codes are not supported on all browsers.

Comment (oswaldoacauan) [reply bigbluebutton-html5/imports/ui/stylesheets/variables/palette.scss]: But you are setting the alpha to 0 thus is the same as `transparent`

File: bigbluebutton-html5/imports/ui/components/user-avatar/styles.scss [hunk 7]
```diff
@@ -49,7 +51,19 @@ $user-list-bg: $color-off-white;
 }
 
 .talking {
-  box-shadow: 0 0 0 1px $user-list-bg, 0 0 0 6px;
+  animation: pulse 1.2s infinite ease-in
```
Comment (Tainan404) [inline bigbluebutton-html5/imports/ui/components/user-avatar/styles.scss]: Why is this animation 1.2s and the other 1s?

File: bigbluebutton-html5/imports/ui/components/audio/audio-controls/styles.scss (hunk 5 above)
Comment (BobakOftadeh) [reply bigbluebutton-html5/imports/ui/components/audio/audio-controls/styles.scss]: The border-radius variable in the general.scss(.2rem) is not the correct value here for this situation.

**Pull request merged**